crates/ra_assists/src/assists/replace_if_let_with_match.rs

```diff
@@ -42,7 +42,7 @@ pub(crate) fn replace_if_let_with_match(ctx: AssistCtx<impl HirDatabase>) -> Opt
         ast::ElseBranch::IfExpr(_) => return None,
     };
 
-    ctx.add_assist(AssistId("replace_if_let_with_match"), "replace with match", |edit| {
+    ctx.add_assist(AssistId("replace_if_let_with_match"), "Replace with Match", |edit| {
```

Comment:
Let's add an assert to `add_assist` method that checks the casing. 

Also, is it correct to captitalize *every* word (as opposed to only the first one)?

Reply:
IntelliJ caps only the first word, which is what I'd expect as well:

![image](https://user-images.githubusercontent.com/1711539/72364600-12e45f80-36f7-11ea-8a37-1df6bf090ed6.png)

Reply:
> Let's add an assert to `add_assist` method that checks the casing.

I'm not sure we want that. For instance we can refernce code in assists like: `Move 'x' to outer scope` or `Extract 'bar' to function`.

Reply:
But the first letter should *always* be a capital one,now?

Reply:
Yep I think that's true.